doc/api/http.md

```diff
@@ -765,6 +765,11 @@ object, so any HTTP response sent, including response headers and payload,
 *must* be written directly to the `socket` object. Care must be taken to
 ensure the response is a properly formatted HTTP response message.
 
+> `err` is an instance of `Error` with two extra columns:
+>
+> + `bytesParsed`: the bytes count of request packet that Node.js may parse correctly;
```

Comment:
s/parse correctly/have parsed incorrectly/?

Reply:
It's the count of 'correctly', not 'incorrectly'. Means Node.js have parsed `bytesParsed` correctly and the left are incorrectly.